Gems/WhiteBox/Code/Source/Viewport/WhiteBoxPolygonGizmoTranslation.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+ 
+#pragma once
+
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <Manipulators/TranslationManipulators.h>
+#include <AzToolsFramework/Viewport/ViewportTypes.h>
+
+namespace WhiteBox
+{
+    class PolygonGizmoTranslation : public AzToolsFramework::TranslationManipulators {
+        public:
+            void ProcessManipulators(const AZStd::function<void(BaseManipulator*)>&) override;
+
+        private:
+            MeshSelection m_selection_manipulator;
+            // ManipulatorViewPolygon m_polygonView;
+            // AzToolsFramework::ManipulatorViewSphere m_vertexView; //!< Manipulator view used to represent a mesh vertex for selection.
+    };
+}
```

Comment:
update: I would not add this, instead, put all the logic in callbacks for the individual manipulators, no need to create a new type an inherit

Have a look at `EditorTransformComponentSelection::CreateTranslationManipulators()` for an example

```
// in EditorWhiteBoxTransformMode

translationManipulators->InstallLinearManipulatorMouseDownCallback(
    [this]([[maybe_unused]] const LinearManipulator::Action& action)
    {
        // do White Box logic here, use `action` which will be how the mouse moved the manipulator
    }
```